### ISSUE TITLE
[master] BLE option fails in Linux

### ISSUE THREAD

**Issue by stronnag** [iNavFlight/inav-configurator#2091]
fails with
```
2024-05-20 @ 20:45:39 -- Error while opening BLE device: TypeError: Cannot read properties of undefined (reading 'requestDevice')
```

As an aside, the ["reference"](https://github.com/electron/electron/tree/v30.0.6/docs/fiddles/features/web-bluetooth) Electron code does work.

**Comment by mrpraline** [comment]
I also cannot connect by BLE (kubuntu 24.04). Windows on the same machine works. I am new to inav, could you tell me where this error message was recorded so I can check if I really have the same problem?

**Comment by stronnag** [comment]
fixed by #2138
